### PR TITLE
manual_correction.py script

### PR DESCRIPTION
**Pull request by valosekj**
Introduced `manual_correction.py` for manual correction. 

Fixes: https://github.com/spine-generic/spine-generic/issues/85

### CODE REVIEW

File: processing/manual_correction.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            "--help",
+            action="help",
+            help="Show this help message and exit."
+        )
+        return parser
+
+
+if __name__ == "__main__":
+    manual_correction = ManualCorrection()
+    manual_correction.main()
```
Comment (jcohenadad) [inline processing/manual_correction.py:139]: add blank line at EOF (PEP8): https://pep8.readthedocs.io/en/release-1.7.x/intro.html

File: processing/manual_correction.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            required=True,
+            metavar="<input yml file>",
+            help="Filename of yml file containing segmentation and vertebral labeling for manual correction."
+        )
+
+        optional = parser.add_argument_group("\nOPTIONAL ARGUMENTS")
+        optional.add_argument(
+            "-o",
+            metavar="<output folder>",
+            help="Path to output folder where manual segmentation and labels will be saved. Default = ../../seg_manual",
```
Comment (valosekj) [inline processing/manual_correction.py:124]: setting path to folder with manual segmentation and labels does not make sense anymore when we have moved these files under a `derivates/ `folder, right @jcohenadad ?

Comment (jcohenadad) [reply processing/manual_correction.py:124]: That’s right: you should instead ask user to input BIDS dataset, and your script will put the files in the appropriate folder (create folder with os.makedirs to have them created recursively if needed)

**Comment by valosekj** [comment]
So, the script should be working now. Please, can you try it?
I tested it from` results/data` folder as follow:
`(venv_sct) valosek@macbook:~/ds002902-download_results/results/data$ python /usr/local/spine-generic/processing/manual_correction.py -i files.yml`

**Comment by valosekj** [comment]
Run also tested from any another directory with `-ifolder` flag:

`(venv_sct) valosek@macbook:~/ds002902-download_results$ python /usr/local/spine-generic/processing/manual_correction.py -i ~/ds002902-download_results/results/data/files.yml -ifolder ~/ds002902-download_results/results/data/`

**Comment by jcohenadad** [comment]
@valosekj what is the status of this? i finished processing some data and could test your PR

**Comment by valosekj** [comment]
@jcohenadad it is ready to use. You can test it. Only `derivatives/label` saving feature is not implemented yet.

**Comment by jcohenadad** [comment]
> Only derivatives/labels saving feature is not implemented yet.

i'll do it (i need it asap)

**Comment by jcohenadad** [comment]
@valosekj sorry i did some butchering on your function, i hope that's ok with you 😅 
i mostly wanted to create APIs, anticipating the move of spine-generic to a proper python package (#96), and avoiding some code duplication down the line.

**Comment by valosekj** [comment]
Looks great 👍

**Pull request merged**